tests/unit/python/execution_tree/primitives/numpy_dtype.py

```diff
@@ -112,6 +112,14 @@ def test_full():
 assert (test_full() == np.full((3, 3), 2, dtype=int)).all()
 
 
+@Phylanx
+def test_full_nodtype():
+    return np.full((3, 3), 2.0)
+
+
+assert (test_full_nodtype() == np.full((3, 3), 2.0)).all()
```

Comment:
I think this test only compare values (not dtypes)

Reply:
Fair point, will fix.